lib/libutils/ext/include/compiler.h

```diff
@@ -276,4 +276,6 @@
 #define fallthrough do {} while (0) /* fallthrough */
 #endif
 
+#define __no_stackprot __attribute__((__optimize__ ("-fno-stack-protector")))
```

Comment:
Please move this just below the `__nostackcheck` line above.

Reply:
It looks like we need to add a conditional for clang, see the failure in the build log.